src/lib/server/orders.ts

```diff
@@ -1617,7 +1646,8 @@ export async function updateAfterOrderPaid(order: Order, session: ClientSession)
 					paidUntil: add(new Date(), { [`${runtimeConfig.subscriptionDuration}s`]: 1 }),
 					createdAt: new Date(),
 					updatedAt: new Date(),
-					notifications: []
+					notifications: [],
+					...(freeProductsById && { freeProductsById })
```

Comment:
```suggestion
					...(!isEmpty(freeProductsById) && { freeProductsById })
```

because `freeProductsById` is initialized wiht an empty object